docs/Using Fleet/MDM-migration-guide.md

```diff
@@ -7,45 +7,173 @@ This section provides instructions for migrating your hosts away from your old M
 1. A [deployed Fleet instance](../Deploying/Introduction.md)
 2. [Fleet connected to Apple](./MDM-setup.md)
 
-## Preparing to migrate manually enrolled hosts
+## Migrate manually enrolled hosts
 
 1. [Enroll](./Adding-hosts.md) your hosts to Fleet with [Fleetd and Fleet Desktop](https://fleetdm.com/docs/using-fleet/adding-hosts#including-fleet-desktop) 
 2. Ensure your end users have access to an admin account on their Mac. End users won't be able to migrate on their own if they have a standard account.
 3. In your old MDM solution, unenroll the hosts to be migrated. MacOS does not allow multiple MDMs to be installed at once.
-4. Send [these guided instructions](#instructions-for-end-users) to your end users to complete the final few steps via Fleet Desktop.
+4. Send [these guided instructions](#how-to-turn-on-mdm) to your end users to complete the final few steps via Fleet Desktop.
     * Note that there will be a gap in MDM coverage between when the host is unenrolled from the old MDM and when the host turns on MDM in Fleet.
 
-## Preparing to migrate automatically enrolled (DEP) hosts
+### End user experience
+
+1. On their **My device** page, once an end user's device is unenrolled from the old MDM solution, the end user will be given the option to manually download the MDM enrollment profile.
+   
+2. Once downloaded, the user will receive a system notification that the Device Enrollment profile has been needs to be installed in the System Settings &gt; Profiles section. 
+   
+3. After installation, the MDM enrollment profile can be removed by the end user at any time.
+
+### How to turn on MDM
+
+1. Select the Fleet icon in your menu bar and select **My device**.
+
+![Fleet icon in menu bar](https://raw.githubusercontent.com/fleetdm/fleet/main/website/assets/images/articles/fleet-desktop-says-hello-world-cover-1600x900@2x.jpg)
+
+2. On your **My device** page, select **Turn on MDM** the button in the yellow banner and follow the instructions. 
+  - If you don’t see the yellow banner or the **Turn on MDM** button, select the purple **Refetch** button at the top of the page. 
+  - If you still don't see the **Turn on MDM** button or the **My device** page presents you with an error, please contact your IT administrator.
+
+<img width="1400" alt="My device page - turn on MDM" src="https://user-images.githubusercontent.com/5359586/229950406-98343bf7-9653-4117-a8f5-c03359ba0d86.png">
+
+## Migrate automatically enrolled (DEP) hosts
 
 > Automatic enrollment is available in Fleet Premium or Ultimate
 
+To migrate automatically enrolled hosts, we will do the following steps:
+
+1. Prepare to migrate hosts
+2. Choose migration workflow and migrate hosts
```

Comment:
I'm going to leave it in because I think it gives the user a heads up that they'll have to choose a migration workflow later. So, if they catch that, they can look ahead at the options.